2023-08-09 00:25:06,665 INFO    MainThread:570746 [internal.py:wandb_internal():90] W&B internal server running at pid: 570746, started at: 2023-08-09 00:25:06.665380
2023-08-09 00:25:06,668 DEBUG   SenderThread:570746 [sender.py:send():236] send: header
2023-08-09 00:25:06,668 INFO    WriterThread:570746 [datastore.py:open_for_write():75] open: /home/mateo/local_phoenix_ws/src/robot_dataset/scripts/wandb/run-20230809_002506-2fi17f34/run-2fi17f34.wandb
2023-08-09 00:25:06,668 DEBUG   SenderThread:570746 [sender.py:send():236] send: run
2023-08-09 00:25:06,852 DEBUG   HandlerThread:570746 [handler.py:handle_request():140] handle_request: check_version
2023-08-09 00:25:06,855 INFO    SenderThread:570746 [dir_watcher.py:__init__():216] watching files in: /home/mateo/local_phoenix_ws/src/robot_dataset/scripts/wandb/run-20230809_002506-2fi17f34/files
2023-08-09 00:25:06,855 INFO    SenderThread:570746 [sender.py:_start_run_threads():815] run started: 2fi17f34 with start time 1691555106
2023-08-09 00:25:06,855 DEBUG   SenderThread:570746 [sender.py:send():236] send: summary
2023-08-09 00:25:06,856 INFO    SenderThread:570746 [sender.py:_save_file():952] saving file wandb-summary.json with policy end
2023-08-09 00:25:06,856 DEBUG   SenderThread:570746 [sender.py:send_request():250] send_request: check_version
2023-08-09 00:25:06,921 DEBUG   HandlerThread:570746 [handler.py:handle_request():140] handle_request: run_start
2023-08-09 00:25:06,934 DEBUG   HandlerThread:570746 [meta.py:__init__():35] meta init
2023-08-09 00:25:06,935 DEBUG   HandlerThread:570746 [meta.py:__init__():49] meta init done
2023-08-09 00:25:06,935 DEBUG   HandlerThread:570746 [meta.py:probe():209] probe
2023-08-09 00:25:06,940 DEBUG   HandlerThread:570746 [git.py:repo():29] git repository is invalid
2023-08-09 00:25:06,940 DEBUG   HandlerThread:570746 [meta.py:_save_pip():53] save pip
2023-08-09 00:25:06,941 DEBUG   HandlerThread:570746 [meta.py:_save_pip():67] save pip done
2023-08-09 00:25:06,941 DEBUG   HandlerThread:570746 [meta.py:probe():247] probe done
2023-08-09 00:25:07,006 DEBUG   SenderThread:570746 [sender.py:send():236] send: files
2023-08-09 00:25:07,007 INFO    SenderThread:570746 [sender.py:_save_file():952] saving file wandb-metadata.json with policy now
2023-08-09 00:25:07,011 DEBUG   HandlerThread:570746 [handler.py:handle_request():140] handle_request: stop_status
2023-08-09 00:25:07,012 DEBUG   SenderThread:570746 [sender.py:send_request():250] send_request: stop_status
2023-08-09 00:25:07,195 DEBUG   SenderThread:570746 [sender.py:send():236] send: telemetry
2023-08-09 00:25:07,405 INFO    Thread-11 :570746 [upload_job.py:push():138] Uploaded file /tmp/tmpn73gki2zwandb/soo5ko6o-wandb-metadata.json
2023-08-09 00:25:07,857 INFO    Thread-7  :570746 [dir_watcher.py:_on_file_created():269] file/dir created: /home/mateo/local_phoenix_ws/src/robot_dataset/scripts/wandb/run-20230809_002506-2fi17f34/files/output.log
2023-08-09 00:25:07,857 INFO    Thread-7  :570746 [dir_watcher.py:_on_file_created():269] file/dir created: /home/mateo/local_phoenix_ws/src/robot_dataset/scripts/wandb/run-20230809_002506-2fi17f34/files/wandb-metadata.json
2023-08-09 00:25:07,857 INFO    Thread-7  :570746 [dir_watcher.py:_on_file_created():269] file/dir created: /home/mateo/local_phoenix_ws/src/robot_dataset/scripts/wandb/run-20230809_002506-2fi17f34/files/requirements.txt
2023-08-09 00:25:07,857 INFO    Thread-7  :570746 [dir_watcher.py:_on_file_created():269] file/dir created: /home/mateo/local_phoenix_ws/src/robot_dataset/scripts/wandb/run-20230809_002506-2fi17f34/files/wandb-summary.json
2023-08-09 00:25:09,856 INFO    Thread-7  :570746 [dir_watcher.py:_on_file_modified():286] file/dir modified: /home/mateo/local_phoenix_ws/src/robot_dataset/scripts/wandb/run-20230809_002506-2fi17f34/files/output.log
2023-08-09 00:25:20,470 WARNING MainThread:570746 [internal.py:wandb_internal():153] Internal process interrupt: 1
2023-08-09 00:25:20,710 WARNING MainThread:570746 [internal.py:wandb_internal():153] Internal process interrupt: 2
2023-08-09 00:25:20,710 ERROR   MainThread:570746 [internal.py:wandb_internal():156] Internal process interrupted.
2023-08-09 00:25:21,013 INFO    WriterThread:570746 [datastore.py:close():279] close: /home/mateo/local_phoenix_ws/src/robot_dataset/scripts/wandb/run-20230809_002506-2fi17f34/run-2fi17f34.wandb
2023-08-09 00:25:21,070 INFO    MainThread:570746 [internal.py:handle_exit():80] Internal process exited
